### PR TITLE
EIP-1: remove requirement for simple summary

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by axic** [comment]
My reasoning: abstract and simple summary kind of is the same thing. For a while there was a suggestion that "simple summary" is a non-technical description and "abstract" is a technical description, but inevitably it results in duplication and a lot of people feel it as a burden without any benefits.

Question: should existing simple summaries removed from merged EIPs?

**Comment by axic** [comment]
cc @nicksavers @Arachnid @karalabe @fjl

File: EIPS/eip-1.md [hunk 1]
```diff
@@ -83,9 +83,8 @@ Other exceptional statuses include:
 
 Each EIP should have the following parts:
 
-- Preamble - RFC 822 style headers containing metadata about the EIP, including the EIP number, a short descriptive title (limited to a maximum of 44 characters), and the author details. See [below](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md#eip-header-preamble) for details.
-- Simple Summary - “If you can’t explain it simply, you don’t understand it well enough.” Provide a simplified and layman-accessible explanation of the EIP.
-- Abstract - a short (~200 word) description of the technical issue being addressed.
+- Preamble - RFC 822 style headers containing metadata about the EIP, including the EIP number, a short descriptive title (limited to a maximum of 44 charac
```
Comment (axic) [inline EIPS/eip-1.md]: Seems I made a typo here and deleted some parts. Will fix in a second.

**Comment by xinbenlv** [comment]
+1

**Comment by nicksavers** [comment]
@axic Can you give examples of where they are redundant? 
And any examples where they actually add value?

**Comment by axic** [comment]
Examples where they are redundant, based on opening random EIPs: 1203, 1538, 1973.

@nicksavers I have yet to find an example where it is useful to have both.

**Comment by nicksavers** [comment]
It's actually pretty nice to have a simple TL;DR of the proposal up front in one or two lines.
The last two examples given are pretty useful to me, but I agree not all are equally useful. I'm not yet convinced. I think it's mostly a deterrent for those that haven't fully understood their own proposal yet.

In [EIP-615](https://eips.ethereum.org/EIPS/eip-615) it's much better in my opinion.

**Comment by axic** [comment]
Well EIP-1 gives no explanation about the difference. When asked, I've kept saying "simple summary is a non-technical description, while abstract is a technical description". Making it mandatory creates some tension and in most of the cases it results in an extra useless section.

I have nothing against leaving it in optionally, but not sure it provides enough value. Why shouldn't just the abstract start out with a single clear sentence explaining what it is?

**Comment by xinbenlv** [comment]
@nicksavers @axic if the purpose is to keep a TL'DR, how about we re-phrase *Abstract* to *Technical Outline* or anything else that makes such an expectation and difference more obvious to authors, how do you like it?

**Comment by Arachnid** [comment]
@xinbenlv I don't think the abstract is intended to describe how the standard 'does it'. It's not clear to me that it's possible to write a clean summary of that in a couple of sentences anyway.

Without looking at the current standard, how would you write a simple summary and abstract for ERC20, for example?

**Comment by xinbenlv** [comment]
@Arachnid:
if I may us ERC-20 as an example

## What this EIP does: 
> A standard interface for tokens.
## How it does
> The following standard allows for the implementation of a standard API for tokens within smart contracts. This standard **provides basic functionality to transfer tokens, as well as allow tokens to be approved* so they can be spent by another on-chain third party.

**Comment by Arachnid** [comment]
@xinbenlv It seems to me that the former makes a good title, while the latter makes a good abstract.

**Comment by xinbenlv** [comment]
@Arachnid agree. I actually think that in most academic articles I've personally seen or in other context, a "Abstract" will usually contain those information. 

If you are proposing that:

* *Simple Summary* -> ***Title***
* *Abstract* -> (remain) ***Abstract***

I am also supporting this idea.

**Comment by Arachnid** [comment]
Well, we already have a title, so this proposal devolves to the same one @axic started with - drop 'simple summary'.

**Comment by xinbenlv** [comment]
Cool, I am for that too.

@nicksavers : WDYT? if you agree, then we have consensus on this issue.

**Comment by axic** [comment]
@nicksavers ping

**Comment by axic** [comment]
@nicksavers what should happen to this one?

**Comment by axic** [comment]
@nicksavers can we merge this or do you have an objection?

**Comment by axic** [comment]
@nicksavers ping

**Comment by axic** [comment]
@Arachnid @nicksavers @Souptacular @gcolvin can this be merged?

**Comment by nicksavers** [comment]
I still think here is a slight difference, but not big enough to break consensus.

**Pull request merged**